docs/user/firststeps.rst

```diff
@@ -102,28 +103,27 @@ directory.  Use the following commands to add data to your project.
 
 .. code-block:: console
 
-    renku dataset create zhbikes
+    renku dataset create flights
     # Output:
     # Creating a dataset ... OK
 
-    renku dataset add zhbikes https://data.stadt-zuerich.ch/dataset/verkehrszaehlungen_werte_fussgaenger_velo/resource/d17a0a74-1073-46f0-a26e-46a403c061ec/download/2019_verkehrszaehlungen_werte_fussgaenger_velo.csv
+    renku dataset add flights https://renkulab.io/gitlab/cramakri/renku-tutorial-flights-data-raw/raw/master/data/v1/2019-01-flights.csv.zip
```

Comment:
I did register this data in Zenodo. I can switch to importing data from there.

Reply:
I think it would be cleaner and easier on the eyes (shorter command). We need a sentence explaining that you can pull in any arbitrary URL and a link to the docs.